clearwater-sip-stress-coreonly.root/usr/share/clearwater/bin/run_stress

```diff
@@ -198,12 +207,12 @@ print "Test complete"
 end_time = datetime.now()
 
 with open(CALLER_STATS) as f:
-	r = DictReader(f, dialect=SIPpstats())
-	for row in r:
-		pass
-	hours, mins, secs, nanoseconds = map(int, row['ResponseTime1(C)'].split(":"))
-	rtt = (nanoseconds / 1000.0) + (secs * 1000.0)
-	print """
+    r = DictReader(f, dialect=SIPpstats())
+    for row in r:
+        pass
```

Comment:
I've probably in theory reviewed this code before, and I know you haven't changed it, but I have no idea what's going on here.